app/livechat/server/lib/Livechat.js

```diff
@@ -1407,3 +1411,7 @@ export const Livechat = {
 settings.watch('Livechat_history_monitor_type', (value) => {
 	Livechat.historyMonitorType = value;
 });
+
+settings.watch('Livechat_enabled', (value) => {
```

Comment:
I see this is a pattern we've been using (just reading the above lines), but there is no value "watching" a setting to the value to somewhere else. using `settings.get(settingName)` is meant for that purpose.

we need to start using `watch` only when we need to trigger additional tasks, or we need to watch for multiple settings to compose a single value. ;)

Reply:
I went with this approach since I was hoping to cache this setting's value in the memory so we don't have to hit the DB to get its value, every time a room gets created. Let me know if you still think it would be better to use settings.get() here instead of settings.watch()

Reply:
I thought so, but the point is that `settings.get()` is already cached, it doesn't hit the DB, that's why I'm suggesting to use it =)

if you look the `settings` variable is an instance of `CachedSettings` class.. 😉  but we should also add a comment on the `settings.get` method, so it is clear to everyone that the value is cached 👍

Reply:
Thanks, I wasn't aware of that. I've updated the code to use `settings.get()` now. 

PS: Also added a comment on the cached setting method so it is more clear that the value is cached